storage-node/src/cache/replacer/lru.rs

```diff
@@ -10,7 +12,8 @@ use super::ReplacerValue;
 /// objects. The replacer will start evicting if a new object comes that makes
 /// the replacer's size exceeds its max capacity, from the oldest to the newest.
 pub struct LruReplacer<K: ReplacerKey, V: ReplacerValue> {
-    cache_map: LinkedHashMap<K, V>,
+    // usize is pin count
+    cache_map: LinkedHashMap<K, (V, usize)>,
```

Comment:
Shall we make `pin_count` a field of `ReplacerValue` since it'll be used by all replacers?

Reply:
Sure! but maybe a github issue and do it later since this pr is too large : (